dcspy/logitech.py

```diff
@@ -152,23 +178,6 @@ def check_buttons(self) -> LcdButton:
         self.lcdbutton_pressed = False
         return LcdButton.NONE
 
-    def check_gkey(self) -> Gkey:
```

Comment:
This is no longer required now as we are listening for the key down/up in the Logitech Gkey callback handler.